packages/eslint-plugin/package.json

```diff
@@ -31,12 +31,14 @@
 	],
 	"main": "index.js",
 	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^4.8.2",
```

Comment:
eslint does a poor job of linting TypeScript. For example:

```ts
export interface WPDataAttachedStore {
	getSelectors: () => WPDataFunctionArray;
	getActions: () => WPDataFunctionArray;
	subscribe: ( listener: () => void ) => () => void;
}
```

With `eslint:recommended`, the above errors with `no-unused-vars` because `listener` is named but not used. However, this is a correctly written interface and `listener` cannot be removed without losing information.

This prompted me to add `plugin:@typescript-eslint/eslint-recommended` [according to documentation](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs).